serfclient/connection.py

```diff
@@ -107,6 +107,14 @@ def handshake(self):
             self._socket = self._connect()
         return self.call('handshake', {"Version": 1}, expect_body=False)
 
+    def auth(self, auth_key):
```

Comment:
Is there any value in making this method public? Would a user ever want to call this directly?